test.sh.log

```diff
@@ -0,0 +1,85 @@
+Running test with user sundar
```

Comment:
Should be ignored, see https://github.com/cu-ecen-aeld/aesd-assignments/blob/master/.gitignore#L8